travis/travis_install_nightly

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-pip install -q QUnitSuite flake8 coveralls pylint Click > /dev/null 2>&1
+pip install -q QUnitSuite flake8 coveralls Click oca-pylint-plugin > /dev/null 2>&1
```

Comment:
@moylop260 IMO `oca-pylint-plugin` is an experimental project. 
I can agree using it it these checks are only informational (write warnings to the log output) and don't affect build status.